src/appkit.cs

```diff
@@ -27689,4 +27861,147 @@ interface NSSharingServicePickerToolbarItemDelegate
 		[Export ("itemsForSharingServicePickerToolbarItem:")]
 		NSObject[] GetItems (NSSharingServicePickerToolbarItem pickerToolbarItem);
 	}
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[Mac (10,16)]
+	[BaseType (typeof(NSToolbarItem))]
+	interface NSSearchToolbarItem
+	{
+		[DesignatedInitializer]
+		[Export ("initWithItemIdentifier:")]
+		IntPtr Constructor (string itemIdentifier);
+
+		[Export ("searchField", ArgumentSemantic.Strong)]
+		NSSearchField SearchField { get; set; }
+
+		[Export ("resignsFirstResponderWithCancel")]
+		bool ResignsFirstResponderWithCancel { get; set; }
+
+		[Export ("minimumWidthForSearchFieldRepresentation")]
+		nfloat MinimumWidthForSearchFieldRepresentation { get; set; }
+
+		[Export ("preferredWidthForSearchFieldRepresentation")]
+		nfloat PreferredWidthForSearchFieldRepresentation { get; set; }
+	}
+
+	delegate NSView NSTableViewDiffableDataSourceCellProvider (NSTableView arg0, NSTableColumn arg1, nint arg2, NSObject arg3);
+	delegate NSTableRowView NSTableViewDiffableDataSourceRowProvider (NSTableView arg0, nint arg1, NSObject arg2);
+	delegate NSView NSTableViewDiffableDataSourceSectionHeaderViewProvider (NSTableView arg0, nint arg1, NSObject arg2);
+
+	[Mac (10,16)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSTableViewDiffableDataSource : NSTableViewDataSource
+	{
+		[Export ("initWithTableView:cellProvider:")]
+		IntPtr Constructor (NSTableView tableView, NSTableViewDiffableDataSourceCellProvider cellProvider);
+
+		[Export ("snapshot")]
+		NSDiffableDataSourceSnapshot<NSObject, NSObject> Snapshot ();
+
+		[Export ("applySnapshot:animatingDifferences:")]
+		void ApplySnapshot (NSDiffableDataSourceSnapshot<NSObject, NSObject> snapshot, bool animatingDifferences);
+
+		[Export ("applySnapshot:animatingDifferences:completion:")]
+		void ApplySnapshot (NSDiffableDataSourceSnapshot<NSObject, NSObject> snapshot, bool animatingDifferences, [NullAllowed] Action completion);
+
+		[Export ("itemIdentifierForRow:")]
+		[return: NullAllowed]
+		NSObject ItemIdentifierForRow (nint row);
+
+		[Export ("rowForItemIdentifier:")]
+		nint RowForItemIdentifier (NSObject identifier);
```

Comment:
`GetRow`

Reply:
hmm... that will be confusing with other API, still it should have a `Get` prefix